hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/BackendEntry.java

```diff
@@ -150,8 +151,7 @@ public byte[] position() {
             }
         }
 
-        public static final class OneColumnIterator
-                            implements BackendColumnIterator {
+        final class OneColumnIterator implements BackendColumnIterator {
```

Comment:
can we keep `static final` mark?

Reply:
static is redundant for inner classes of interfaces. 

![static](https://user-images.githubusercontent.com/13429553/165522755-908eaacd-83fb-40cf-a648-5f9ea36758fd.JPG)

https://beginnersbook.com/2016/03/nested-or-inner-interfaces-in-java/#:~:text=Nested%20interfaces%20are%20static%20by,inside%20interface%20is%20public%20implicitly.